src/components/SystemHealthIcon/index.tsx

```diff
@@ -7,7 +7,7 @@ import './index.scss';
 
 type SystemHealthIconProps = {
   className?: string;
-  status: IconStatus;
+  status?: IconStatus;
```

Comment:
Why was this changed to be optional?

Reply:
Ah I should have added a TODO comment or removed it.  The Figma mockups show a lot more health statuses than just the IconStatus type.  We haven't solidified what those are/can be yet based on CEDAR responses.  I think I had added that during development to catch a default, but the IconStatus types might need to be reviewed once we understand more about CEDAR statuses in relation to relevant health icons

Reply:
Gotcha. How about we keep `status` as non-optional in this PR, and add a TODO on the `IconStatus` type noting that it might need to change based on the Figma mockups/CEDAR responses?

Reply:
Will do, thanks